templates/zerver/help/import-your-settings.md

```diff
@@ -0,0 +1,34 @@
+# Import your settings
+
+When you create a new account using an email address already associated with a
+Zulip account in another organization, the registration form will offer to import
+the user settings from one of your existing account profiles.
+
```

Comment:
This sentence is a bit hard to process; let's try something like this:

> When you create a Zulip account using an email address already associated with an account in another Zulip organization, you can import your user settings from your existing account.

Reply:
This is not quite accurate, since (I assume) you can only import settings between Zulip Cloud orgs or orgs that are on the same server. We could try to add a warning block to this effect, but maybe that's too complicated? @timabbott for your thoughts.

Reply:
We could also consider adding a benefits sentence (i.e. why importing your settings is nice) here.

Reply:
@alya How about adding something like, "It's the quickest and easiest way to preserve the user settings that you've already customized to your liking."?

Reply:
Mm, riffing off that, maybe something like this?

"It's a convenient way to preserve the user settings that you've already customized."